packages/react-menu/jest.config.js

```diff
@@ -1,9 +1,18 @@
-const { createConfig, resolveMergeStylesSerializer } = require('@fluentui/scripts/jest/jest-resources');
-const path = require('path');
-
-const config = createConfig({
-  setupFiles: [path.resolve(path.join(__dirname, 'config', 'tests.js'))],
-  snapshotSerializers: [resolveMergeStylesSerializer()],
-});
-
-module.exports = config;
+/**
+ * @type {jest.InitialOptions}
```

Comment:
For this to show errors in the editor you'll also need `// @ts-check` at the top of the file (same in other JS files with types)

Reply:
I guess my perspective is that the point of adding type annotations is mainly to give a level of error checking while editing (even if CI doesn't check the files), since that's better than nothing. With annotations but no ts-check, it's a bit better documented and has intellisense, but you still don't get any notification in the editor if you misspell something.